src/SFA.DAS.EmployerApprenticeshipsService.Application/Queries/GetTheRdsRequiredStatistics/GetTheRdsRequiredStatisticsHandler.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MediatR;
+using SFA.DAS.EAS.Account.Api.Types;
+using SFA.DAS.EAS.Domain.Data.Repositories;
+
+namespace SFA.DAS.EAS.Application.Queries.GetTheRdsRequiredStatistics
+{
+    public class GetTheRdsRequiredStatisticsHandler : IAsyncRequestHandler<GetTheRdsRequiredStatisticsRequest, GetTheRdsRequiredStatisticsResponse>
```

Comment:
The request/response/handler can be called `GetStatistics...` since the endpoint isn't necessarily RDS specific.